File: 0001_0100/0060/0060.c
#include <stdio.h>
#include <math.h>

int issim(int a)
{
	if (a == 2)
		return 1;
	if (a == 1 || a % 2 == 0)
		return 0;
	for (int i = 3; i <= sqrt(a); i += 2)
		if (a % i == 0)
			return 0;
	return 1;
}

int main()
{
	FILE *f = fopen("input.txt", "r"), *q = fopen("output.txt", "w");
	int n, m = 0, i, k = 1;

	fscanf(f, "%d", &n);
	for (i = 3; ; i += 2)
		if (issim(i) && issim(++k) && ++m == n)
			break;
	fprintf(q, "%d", i);
	return 0;
}

File: 0001_0100/0060/0060.go
package main

import (
	"fmt"
	"math"
	"os"
)

func prime(n int) bool {
	if n == 2 {
		return true
	} else if n == 1 || n%2 == 0 {
		return false
	}
	for i, s := 3, int(math.Sqrt(float64(n))); i <= s; i += 2 {
		if n%i == 0 {
			return false
		}
	}
	return true
}

func main() {
	f, _ := os.Open("input.txt")
	q, _ := os.Create("output.txt")
	var n, m int
	fmt.Fscan(f, &n)
	i := 3
	for k := 1; ; i += 2 {
		if prime(i) {
			k++
			if prime(k) {
				m++
				if m == n {
					break
				}
			}
		}
	}
	fmt.Fprint(q, i)
}
